packages/esm-patient-search-app/src/compact-patient-search/compact-patient-banner.scss

```diff
@@ -19,13 +20,40 @@
   }
 }
 
+.deceased.patientSearchResult {
+  background-color: colors.$gray-80;
+
+  .patientName {
+    color: $ui-01;
+  }
+
+  .demographics,
+  .patientIdentifiers,
+  .identifierTag {
+    color: $ui-02;
+
+    &:hover,
+    &:focus {
+      background-color: colors.$gray-70;
+      border: 0;
+      outline: 0;
+    }
```

Comment:
Not sure we need these bits. Why do we want to highlight demographics and identifiers with a grey background on hover and focus?